package.json

```diff
@@ -14,8 +14,8 @@
 		"url": "https://github.com/woocommerce/woocommerce/issues"
 	},
 	"scripts": {
-		"build": "pnpm exec turbo run turbo:build -- -- ",
-		"test": "pnpm exec turbo run turbo:test -- --",
+		"build": "pnpm exec turbo run turbo:build",
```

Comment:
we have to remove these extra `--` for the turbo scripts to function in v7. they were to handle an issue with arg parsing in pnpm 6